scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -508,9 +510,35 @@ def get_tests_list(self):
                             testcases.remove(case.detailed_name)
         return testcases
 
-    def add_testsuites(self, testsuite_filter=None):
+    def _is_testsuite_selected(self, suite: TestSuite, testsuite_filter, testsuite_patterns_r):
```

Comment:
this name of that method suggests that testsuite is selected by the user ... but in implementation is that testsuite is not selected directly by the user or filters are not enabled. 

Please rename it or change implementation

Reply:
"filtering" is to be confused with all the filtering activities that follow later, I prefer to use other language here, be cause here we including, not excluding.